src/model/entity/entity.go

```diff
@@ -1 +1,197 @@
 package entity
+
+import (
+	"time"
+)
+
+const RoomsInWidth, RoomsInHeight = 3, 3
+const RoomsNum = RoomsInWidth * RoomsInHeight
+
+const MaxPassageParts = 3
+const MaxPassagesNum = RoomsNum - 1
+
+const MaxMonstersPerRoom = 2
+const MaxConsumablesPerRoom = 3
+
+type LevelNum uint
+
+const (
+	One LevelNum = iota
+	Two
+	Three
+	Four
+	Five
+	Six
+	Seven
+	Eight
+	Nine
+	Ten
+	Eleven
+	Twelve
+	Thirteen
+	Fourteen
+	Fifteen
+	Sixteen
+	Seventeen
+	Eighteen
+	Nineteen
+	Twenty
+	TwentyOne
+)
+
+type Dimension uint
+
+const (
+	X Dimension = iota
+	Y
+	CoordinatesNum
+)
+
+type Measure uint
+
+const (
+	Height Measure = iota
+	Width
+	MeasuresNum
+)
+
+type Statistics uint
+
+const (
+	HealthStat Statistics = iota
+	AgilityStat
+	StrengthStat
+)
+
+type EnemyType uint
+
+const (
+	Zombie EnemyType = iota
+	Vampire
+	Ghost
+	Ogre
+	Snake
+)
```

Comment:
Вот тут всё правильно. Но не хватает полного нейминга, чтобы не возникало конфликта имён.

```suggestion
type EnemyType uint

const (
	EnemyTypeZombie EnemyType = iota
	EnemyTypeVampire
	EnemyTypeGhost
	EnemyTypeOgre
	EnemyTypeSnake
)
```